src/Spanner/Database.php

```diff
@@ -309,7 +309,9 @@ public function create(array $options = [])
             'statements' => [],
         ];
 
-        $statement = sprintf('CREATE DATABASE `%s`', DatabaseAdminClient::parseDatabaseFromDatabaseName($this->name));
+        $databaseNameComponents = DatabaseAdminClient::parseName($this->name());
```

Comment:
Small nit: I have no issue with direct access of the returned result from `parseName`, ex. 

```php
$databaseName = DatabaseAdminClient::parseName($this->name())['database'];
```